### DISCUSSION TITLE
Quinidine PBPK model for CYP2D6, CYP3A4 and P-gp drug-drug(-gene) interaction predictions

### DISCUSSION THREAD

**Discussion by DeniTue** [Open-Systems-Pharmacology/Forum#1543] [Announcement]
Dear all,

We have been working on a parent-metabolite PBPK model of quinidine and 3-hydroxyquinidine, evaluated within a comprehensive drug-drug(-gene) interaction network as (i) CYP2D6 perpetrator and (ii) CYP3A4 and P-gp victim, including a total of 13 compounds.

The corresponding manuscript has been published in CPT: PSP and the model snapshots have been uploaded as OSP repositories to GitHub.

Related publication:
Feick D, Rüdesheim S, Marok FZ, Selzer D, Loer HLH, Teutonico D, Frechen S, van der Lee M, Moes DJAR, Swen JJ, Schwab M, Lehr T. Physiologically-based pharmacokinetic modeling of quinidine to establish a CYP3A4, P-gp, and CYP2D6 drug–drug–gene interaction network. CPT Pharmacometrics Syst Pharmacol 2023. DOI: [10.1002/psp4.12981](https://doi.org/10.1002/psp4.12981).

All models can be found [here]( https://github.com/orgs/Open-Systems-Pharmacology/repositories).

Best regards,
Denise

**Comment by prvmalik** [comment]
Thanks Denise, I'm really excited to try it out!
